tools/rna_tools/intarna/intarna.xml

```diff
@@ -1,4 +1,4 @@
-<tool id="intarna" name="IntaRNA" version="3.1.1">
+<tool id="intarna" name="IntaRNA" version="3.1.2">
```

Comment:
if you want to save your work in the long run, this version could also be macrofied ... so you only need to bump it once